app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(name="User[username]", name ="User[email]", name ="User[password]")
+    @user = User.new(user_params)
+    @user.save
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
```

Comment:
- [ ] Create an `update` action that is similar to the `create` action, except it will redirect to the `edit_user_path` if it succeeds or `render :edit` if it doesn't. It will handle what will happen if the user is successful in editing the profile or not, please check my comment in your `routes` file if you are seeing this first.